src/win32_init.c

```diff
@@ -40,7 +40,7 @@ static const GUID _glfw_GUID_DEVINTERFACE_HID =
 #if defined(_GLFW_USE_HYBRID_HPG) || defined(_GLFW_USE_OPTIMUS_HPG)
 
 #if defined(_GLFW_BUILD_DLL)
- #warning "These symbols must be exported by the executable and have no effect in a DLL"
+ #pragma message("These symbols must be exported by the executable and have no effect in a DLL")
```

Comment:
Would `#pragma warning` be better in that case?

Reply:
`#pragma warning` is used for a different purpose. See [MS docs](https://docs.microsoft.com/en-us/cpp/preprocessor/warning?view=msvc-160). Prefixing the message with `"Warning: "` might be enough.

Reply:
I see, thank you for the info.